source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRErrors.h

```diff
@@ -19,6 +19,13 @@ typedef NS_ENUM(NSUInteger, ACRInputError)
     ACRInputErrorGreaterThanMaxDate,
 };
 
+typedef NS_ENUM(NSUInteger, ACRRenderingError)
```

Comment:
>ACRRenderingError [](start = 28, length = 17)

Minor: I would call this ACRRenderingStatus or ACRRenderingResult, since they aren't all errors.

Reply:
thanks; make sense

---
In reply to: [321498435](https://github.com/microsoft/AdaptiveCards/pull/3354#discussion_r321498435) [](ancestors = 321498435)